components/Welcome.jsx

```diff
@@ -1,7 +1,23 @@
 import React from "react";
+import Image from "next/image";
 
 const Welcome = () => {
-  return <div>Welcome</div>;
+  return (
+    <div>
+      <h1 className="text-[#0F121A] text-center font-bold xl:text-7xl l:text-6xl s:text-7xl text-5xl">
```

Comment:
use acm colors so like `text-acm-blue` you can refer to the tailwind.config.js file for the color mappings